src/main/kotlin/com/bridgecrew/ui/CodeDiffPanel.kt

```diff
@@ -84,7 +84,6 @@ class CodeDiffPanel(val result: BaseCheckovResult, private val isErrorBubble: Bo
                 var currentLine = (block[0] as Double).toInt()
                 val code = block[1]
                 codeBlock += "$currentLine\t$code".replace("\n", "")
-                currentLine++
```

Comment:
I think the fix you wanted here is to take the currentLine definition out of the foreach, not remove the ++